service/history/mutableStateBuilder.go

```diff
@@ -81,6 +81,7 @@ type (
 
 		executionInfo    *persistence.WorkflowExecutionInfo // Workflow mutable state info.
 		replicationState *persistence.ReplicationState
+		versionHistories persistence.VersionHistories
```

Comment:
plz change this to a pointer so we can use `null`

Reply:
Done